src/components/CountryPerformanceOnRisk.js

```diff
@@ -73,9 +73,13 @@ export class CountryPerformanceOnRisk extends Component {
 
 
   updateValue(idxOfSelector, selectedCountry) {
-    selectedCountry = selectedCountry || { value: "" }
-    this.props.dispatch(countryIsSelected(idxOfSelector, selectedCountry.value))
-    this.props.dispatch(fetchData(selectedCountry.value,this.props.views[1].risk))
+    if(!selectedCountry || selectedCountry.value === "") {
+      this.props.dispatch(countryIsSelected(idxOfSelector, ""))
+    } else {
+      this.props.dispatch(countryIsSelected(idxOfSelector, selectedCountry.value))
+      this.props.dispatch(fetchDataIfNeeded(selectedCountry.value,this.props.views[1].risk))
```

Comment:
Same here, but think this need to think more carefuly here as in this case we need to know which view to work on...